### DISCUSSION TITLE
OBS 27.2 logo has black square on Lubuntu 20.04.3

### DISCUSSION THREAD

**Discussion by TotalCaesar659** [obsproject/obs-studio#5964] [Development Questions] [answered]
On Lubuntu 20.04.3, LXQt menu shows OBS logo with black square.
![0](https://user-images.githubusercontent.com/14265316/154079707-227913bd-164f-45d7-a193-96eeb94a0edb.png)
When opening [this file](https://github.com/obsproject/obs-studio/blob/master/UI/xdg-data/icons/obs-logo-scalable.svg) in LXImage, it shows black square too.
![0](https://user-images.githubusercontent.com/14265316/154079825-20218873-b029-4b0d-ab4e-83da634787a9.png)
OBS 27.1.3 logo was alright before changing it to this SVG.

**Comment by ElectronicWar** [comment]
How did you install OBS on your system?
Official PPA? Flatpack? Another method?

**Comment by TotalCaesar659** [comment]
Official PPA.

**Comment by Fenrirthviti** [comment, marked as answer]
Discussions is not the correct place to report potential bugs, I would prefer if this went through the normal support channels to validate that it's not an environment/configuration issue.

Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
